src/pages/stake/components/StakedCard.tsx

```diff
@@ -37,7 +37,8 @@ const StakedCard = (props: PropsWithChildren<Props>) => {
           style={{ justifyContent: "space-between", alignItems: "flex-end" }}
         >
           <span className={styles.value}>
-            {currency.symbol} <Read amount={value} decimals={0} fixed={2} />
+            {currency.symbol}{" "}
+            <Read amount={value} denom="" decimals={0} fixed={2} />
```

Comment:
Decimals included.  May want to just have `fixed` default to 2 in the `Read` component

Reply:
this also for a currency amount, not a token denom.